app/views/mailers/users/rdv_mailer/rdv_cancelled_by_user.html.slim

```diff
@@ -0,0 +1,9 @@
+div
+  p Bonjour,
+  p Votre RDV #{@rdv.motif.service.name} du #{l(@rdv.starts_at, format: :human)} a bien été annulé.
+
+  - if @rdv.lieu.present? && @rdv.reservable_online?
+    p Vous pouvez reprendre un rendez-vous en cliquant sur le lien ci-dessous.
+
+    .btn-wrapper
+      = link_to 'Reprendre RDV', lieu_url(@rdv.lieu.id, search: {departement: @rdv.organisation.departement, motif: @rdv.motif.name, service: @rdv.motif.service.id, where: @rdv.address}), class:'btn btn-primary'
```

Comment:
je ne crois pas que ce soit nouveau mais ce lien a l'air cassé : `LieuxController#show
errorundefined method `match' for nil:NilClass ` il faudrait créer un ticket de bug a part pour investiguer

Reply:
Ah oui, sur une expression rationnelle, c'est presque pas surprenant :smile: 

Il y a une blague, que j'ai pas entendu depuis longtemps, qui dit: 
> Quand tu résous un problème avec une expression rationnelle, tu crées un deuxième problème
:rofl: 

Je vais regarder dans ce ticket

Reply:
J'ai corrigé le soucis du lien, par contre il reste un problème : quand tu arrives directement par le lien tu n'as pas les champs pré-rempli au dessus.

Je note ça dans un fichier TODO, et je verrais peut-être à en faire un ticket dans la journée.

![Capture d’écran de 2020-12-03 11-54-24](https://user-images.githubusercontent.com/42057/101000519-5093e600-355e-11eb-8f9e-60fe0d9f59d6.png)